proposals/lip-0002.md

```diff
@@ -112,17 +112,17 @@ We believe that given these numbers, 15KB is the most adequate choice in terms o
 
 4. Update `modules.blocks.process.generateBlock`:
 
-	- Call `modules.transactions.getUnconfirmedTransactionList` without a limit to get all the unconfirmed transactions. As those transactions should always be approximately equal to 15KB.
+	- Call `modules.transactions.getUnconfirmedTransactionList` without a limit to get all the unconfirmed transactions. As those transactions should always be approximately equal to 15 KiB.
 
-5. Update `constants.maxPayloadLength` to 15KB.
+5. Update `constants.maxPayloadLength` to 15 KiB.
 
 6. Remove `constants.maxTxsPerBlock` from `constants.js`.
 
 7. Remove the redundant `numberOfTransactions` check from blocks verification for historical data, as it is already included in block's signature.
 
-8. Remove the redundant `payloadLength` check from blocks verification for historical data (up to 1048576KB), as it is already included in block's signature.
+8. Remove the redundant `payloadLength` check from blocks verification for historical data (up to 1048576 KiB), as it is already included in block's signature.
```

Comment:
Here I'm not sure if changing the unit without changing the value is correct. I actually can't recall right now where the value comes from.

Reply:
I just remove this value, as it is about removing the check and the value is not important for the specifications.